s2spy/rgdr/label_alignment.py

```diff
@@ -358,15 +357,14 @@ def _rename_datasets(
         A list of the input clustered data, with the labels renamed.
     """
     renamed = []
-    for split, _rgdr in enumerate(rgdr_list):
+    for split, _ in enumerate(rgdr_list):
         # A copy is required to not modify labels of the input data
         data = copy(clustered_data[split])
-        labels = data["cluster_labels"].values
-        i_interval = str(_rgdr.cluster_map["i_interval"].values)  # type: ignore
+        labels = data["cluster_labels"].values.astype("U32")
```

Comment:
In xarray you cannot have arbitrarily long strings as coordinates, so I chose for a 32 character Unicode string, as that would fit well enough if users would want to rename clusters to something more descriptive like "north-west pacific ocean".

Otherwise the name would be trimmed if users try to change values in the array. E.g.:
![image](https://user-images.githubusercontent.com/12114825/211825640-1259ebc6-c072-4a42-986a-1b10f24801fe.png)